app/celery/scheduled_tasks.py

```diff
@@ -42,6 +42,11 @@
 )
 from app.notifications.process_notifications import send_notification_to_queue
 from app.v2.errors import JobIncompleteError
+from celery import Task
+
+# https://stackoverflow.com/questions/63714223/correct-type-annotation-for-a-celery-task
+save_smss = cast(Task, save_smss)
```

Comment:
It seems wrong to me to need to add a library into our production code to make vscode give correct type hints in python 🤷